Foundation/NSHost.swift

```diff
@@ -14,8 +14,19 @@ import Darwin
 import Glibc
 #endif
 
+import Foundation
 import CoreFoundation
 
+extension String {
+    public static func fromCString(_ cString: UnsafeMutablePointer<Int8>) -> String {
```

Comment:
I'm confused by the use of this function below. Is there a reason you can't use `String.init(cString:)` or `String.init(bytesNoCopy:length:encoding:freeWhenDone:)`?